docs/user-guide/src/bmo/automatic_secure_boot.md

```diff
@@ -0,0 +1,30 @@
+# Automatic secure boot
+
+The automatic secure boot feature allows enabling and disabling UEFI (Unified Extensible Firmware Interface) secure boot when provisioning a host. This feature requires supported hardware and compatible OS image. The current hardwares that support enabling UEFI secure boot are `iLO`, `iRMC` and `Redfish` drivers.
+
+Check also
+
+- [Ironic UEFI secure boot](https://docs.openstack.org/ironic/latest/admin/security.html#uefi-secure-boot-mode)
+- [Wikipedia UEFI secure boot](https://en.wikipedia.org/wiki/UEFI#SECURE-BOOT)
+
+## Advantages/Benefits
+
+We need the automatic secure boot when provisioning a host with high security requirements. Based on checksum and signature, the secure boot protects the host from loading malicious code in the boot process before loading the provisioned operating system.
+
+## Usage
```

Comment:
and maybe:
```suggestion
## How to use it
```

Reply:
same https://github.com/metal3-io/metal3-docs/pull/282#discussion_r982022576